src/main/java/org/cascadebot/cascadebot/commands/fun/RandomNumberCommand.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+ 
+package org.cascadebot.cascadebot.commands.fun;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import java.io.IOException;mport java.util.Random;
+
+
+public class RandCommand extends ICommandMain
```

Comment:
As per my other suggestions, perhaps change the name of this command to "RandNumCommand".